package.json

```diff
@@ -314,23 +322,72 @@
 						},
 						"eye": {
 							"type": "number"
+						},
+						"play": {
+							"type": "number"
+						},
+						"star": {
+							"type": "number"
 						}
 					},
 					"default": {
-						"fourPointStar": 1,
-						"chevron": 1,
-						"threePointStar": 1,
+						"ex": 1,
+						"fox": 1,
+						"wing": 1,
 						"hole": 1,
 						"frame": 1,
 						"curve": 1,
-						"eye": 1
+						"eye": 1,
+						"play": 1,
+						"star": 1
 					},
 					"additionalProperties": false
 				},
 				"cursorless.individualHatAdjustments": {
 					"description": "Separate adjustments for each hat shape",
 					"type": "object",
-					"default": {}
+					"default": {
```

Comment:
Put a default value in to make this one easier to use when tweaking.  Tempted to remove this stuff but it is super useful